src/main/java/be/minelabs/data/MinelabsLangProvider.java

```diff
@@ -41,14 +44,14 @@ public CompletableFuture<?> run(DataWriter writer) {
             Objects.requireNonNull(key);
             Objects.requireNonNull(value);
 
-            if (translationEntries.containsKey(key) && !translationEntries.get(key).isEmpty()) {
+            if (translationEntries.containsKey(key) &&
+                    !translationEntries.get(key).isEmpty() && !translationEntries.get(key).equals(EMPTY_TRANSLATION)) {
```

Comment:
Nice catch to see this had to be overriden for the extra check here.

I believe there is another way in which the override can be avoided. If you first add the existing translation file (should not give duplicates) and then add all the defaults inside of a try catch. Then just silently ignore the catch because that just means the value wasn't missing and indeed it doesn't need to be added to the file anymore. What do you think?

Reply:
The solution you are suggesting still produces an error. 
Loading a file just iterates over all entries and adds them to the map.
Everytime you add something to the map the check for duplicates is performed.

So loading the file and then adding them still produces the error.
Which is why I override the methode to allow duplicate keys ONLY if the value is empty or matches the empty_translation string.

Reply:
That's what the try catch is for.

It also serves as an extra check against duplicate values in the file, which need to be avoided :).

Reply:
That would introduces a dozen of try and catches that we completly ignore.
You either get spammed by the duplicate warning (if you allow a `printStackTrace` ) or you don't see any warning at all.

I can see this being handy for when we know the value exitst and we want to override it like for the DutchLangProvider.
But otherwise this solution does not look good.